AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DatabaseErrorDialog.java

```diff
@@ -270,12 +272,25 @@ public MaterialDialog onCreateDialog(Bundle savedInstanceState) {
                         })
                         .show();
             }
+            case DIALOG_DB_LOCKED: {
+                //If the database is locked, all we can do is ask the user t exit.
```

Comment:
```suggestion
                //If the database is locked, all we can do is ask the user to exit.
```

As long as I have one textual nit that's important might as well throw in the other one I would have otherwise ignore ;-)

I think this was the last commit in the stack, I'd like to rebase not squash so if you could merge this in (ignoring the fact that github makes them committable - it's just a nice presentation to see my suggestion - that'd be `<kiwi>legend</kiwi>`

Reply:
I'm used to rebasing commits in the middle of the stack now, so it's not a bother. Both were at the end